### PR TITLE
update locale files for zh-TW and zh-CN

### PR DESCRIPTION
**Pull request by kaochenlong**
fix for missing translation issues both in zh-TW and zh-CN locale

### CODE REVIEW

**Comment by dwradcliffe** [comment]
Thanks for the pull request! We usually like to get at least one review before merging translation changes. (I'm unqualified to review this 😄)

**Comment by kaochenlong** [comment]
sure, no problem :)

File: config/locales/zh-CN.yml [hunk 1]
```diff
@@ -92,7 +92,7 @@ zh-CN:
         uptime: 服务状况
       header:
         dashboard: 控制台
-        search_gem: 搜索 Gems&hellip;
```
Comment (franklinyu) [inline config/locales/zh-CN.yml]: I prefer the original one here. See https://github.com/rubygems/rubygems.org/pull/1803#issuecomment-431605597

**Comment by sonalkr132** [comment]
Hi, it seems the existing translation is preferred for `zh-CN` and `zh-TW` change was updated in a different PR. I am closing this.
Thank you for taking the time to send pull request anyway :100:

**Pull request closed without merging**